api/gin/webServer.go

```diff
@@ -281,6 +281,10 @@ func (ws *webServer) Close() error {
 	err := ws.httpServer.Close()
 	ws.Unlock()
 
+	if err != nil {
+		log.Warn("cannot close http server", "error", err)
```

Comment:
or, we can wrap the error here (since it is already logged)
like
```GO
if err != nil {
    err = fmt.Errorf("%w while closing the http server in gin/webServer", err)
}
```

Reply:
done